### ISSUE TITLE
[18.12.19]Distributed Representations ofWords and Phrases and their Compositionality

### ISSUE THREAD

**Issue by strutive07** [SSUHan/PaparReviews#11]
# [Convolutional Neural Networks for Sentence Classification](https://arxiv.org/abs/1408.5882)
- EMNLP 2014
- Yoon Kim

New York University

## Keywords
-  CNN, textCNN

## Contribution 
- 기존 Vision 분야에서 많이 사용하던 CNN 모델을 nlp에도 적용하였고, 간단한 레이어도 좋은 성능을 낼 수 있다고함.
- 
 
## Proposed Architecture
![image](https://user-images.githubusercontent.com/26921984/50196163-7d588e00-0385-11e9-9d4f-be9d15ec77db.png)
![image](https://user-images.githubusercontent.com/26921984/50196175-88132300-0385-11e9-8673-5721ac969630.png)
h : window size
b : bias
w : filter

Regularization

dropout
dense 직전, output직전
![image](https://user-images.githubusercontent.com/26921984/50196435-6f573d00-0386-11e9-9152-77d25d65ede3.png)

l2 loss

![image](https://user-images.githubusercontent.com/26921984/50196794-ed681380-0387-11e9-8d17-b59272b575b7.png)

## Dataset
![image](https://user-images.githubusercontent.com/26921984/50196813-ff49b680-0387-11e9-8ec3-ecba573488c8.png)
![image](https://user-images.githubusercontent.com/26921984/50196814-02dd3d80-0388-11e9-91ce-51cda553b936.png)

![image](https://user-images.githubusercontent.com/26921984/50196804-f658e500-0387-11e9-898f-c829f5800bfa.png)

Pre-trained word2vec 사용

![image](https://user-images.githubusercontent.com/26921984/50196826-19839480-0388-11e9-9a49-8f13a6df74ee.png)
![image](https://user-images.githubusercontent.com/26921984/50196839-2a340a80-0388-11e9-864c-cb9bfe08002b.png)

Multichannel vs. Single Channel Models
multichannel 을 사용하는게 single channel 보다 overfitting을 방지하고, 특히 적은 데이터셋에서 더 좋은 성능을 예상하였으나, 테스트 결과 multichannel 모델이 모든 데이터셋에서 좋은 성능을 보이지 않는다.

Static vs. Non-static Representations
기존의 word2vec가 학습됨으로써 단어간에 좀 더 의미있는 것끼리 cosine similarity가 향상됨을 알수있다.기존 pre-train된 static word2vec에서는 good은 great과 유사도가 가장 높은데, Non-static으로 word2vec를 학습 후 nice와 더 가까운것을 알 수 있다. 

## Valuable Relative Works
- 
-
